arch/eventloop_posix_tcp.c

```diff
@@ -778,7 +779,42 @@ TCP_openActiveConnection(UA_ConnectionManager *cm, const UA_KeyValueMap *params,
                        "TCP\t| Lookup of %s failed (%s)",
                        hostname, gai_strerror(error));
 #endif
+
```

Comment:
The EventLoop encapsulates architecture- and OS-specific differences.
This file does TCP and nothing else.

please handle the domain-name rewriting business one layer up before you call into the EventLoop layer.

I thought mdns was udp based. Why changes in the TCP layer?